src/libraries/pkg/test/testPackages.proj

```diff
@@ -157,6 +157,7 @@
 
     <MakeDir Directories="$(TestArchiveTestsRoot)" />
     <ZipDirectory
+        Condition="'@(SupportedPackage)' != ''"
```

Comment:
Should we check the target branch also to start with 'release/'? I don't want this to silently pass on master if somehow we break producing packages and then the PR build won't fail.